docs/reference/search/explain.asciidoc

```diff
@@ -118,15 +112,13 @@ This will yield the same result as the previous request.
 `_source`::
 
     Set to `true` to retrieve the `_source` of the document explained. You can also
-    retrieve part of the document by using `_source_include` & `_source_exclude` (see <<get-source-filtering,Get API>> for more details)
+    retrieve part of the document using `_source_include` & `_source_exclude` (see <<get-source-filtering,Get API>> for more details)
 
 `stored_fields`::
-    Allows to control which stored fields to return as part of the
-    document explained.
+    Allows to control which stored fields to return as part of the explained document.
 
```

Comment:
"Allows to"  is one of my least-favorite patterns in the docs. I'd opt for "Controls which stored fields are returned with the explanation." Or maybe just "The stored fields to return with the explanation."